### PR TITLE
made navbar and header w react, change->bootstrap

### PR DESCRIPTION
**Pull request by prewe5294**
Made nav bar and header with react, changed to use bootstrap

### CODE REVIEW

File: client/package-lock.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
         "@testing-library/react": "^13.4.0",
         "@testing-library/user-event": "^13.5.0",
         "axios": "^1.1.2",
+        "beets": "file:..",
```
Comment (piratecody) [inline client/package-lock.json:15]: Do you know why this was added?

Comment (prewe5294) [reply client/package-lock.json:15]: I was having issues getting this file up to date with the main, and it let me merge any changes you guys made but it didn't let me remove the things that I think you guys deleted. I tried to make sure the files were all the same as before but some might have slipped past me. Let me know what needs to be changed from those files to make them correct and I'll do that before this pull request goes through

Comment (prewe5294) [reply client/package-lock.json:15]: both package-lock.json files may be wrong

Comment (piratecody) [reply client/package-lock.json]: Awesome, looks good

**Review by piratecody — APPROVED** [review]

**Pull request merged**